index.js

```diff
@@ -1,12 +1,28 @@
 const express = require("express");
 const app = express();
+const mongoose = require("mongoose");
+const config = require("./config");
+
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+mongoose
+    .connect(config.MONGO, {
```

Comment:
This is what I am saying. `MONGO` is a `env` variable that you have used in the `config.js` file. We need to have a template and let the users update this field for themselves.

Reply:
I tried creating a .env file, but the npm package "dotenv" was not working in the container. So the config.js was a workaround